lib/src/entrypoint.dart

```diff
@@ -139,6 +139,16 @@ class Entrypoint {
   /// The path to the entrypoint package's pubspec.
   String get pubspecPath => p.normalize(root.path('pubspec.yaml'));
 
+  /// Whether the entrypoint package's pubspec contains overrides from an
+  /// overrides files.
+  ///
+  /// If this is `true`, [pubspecOverridesPath] contains the path to the
+  /// overrides file.
+  bool get hasPubspecOverrides => pubspecOverridesPath != null;
+
+  /// The path to the entrypoint package's pubspec overrides file.
+  String? get pubspecOverridesPath => root.pubspec.overridesLocation?.path;
```

Comment:
The path is the same regardless of whether or not it exists... Better to have a boolean saying if it exists, and always have the path be the same.

Reply:
Done